openipam/api/serializers/base.py

```diff
@@ -65,6 +65,7 @@ def validate(self, value):
         if value:
             try:
                 value = EUI(str(value), dialect=mac_bare)
+                return
```

Comment:
I don't think this changes anything here, is there an issue this is supposed to resolve?